etcdctl/ctlv3/command/printer.go

```diff
@@ -221,6 +221,8 @@ func makeEndpointStatusTable(statusList []epStatus) (hdr []string, rows [][]stri
 			status.Resp.StorageVersion,
 			humanize.Bytes(uint64(status.Resp.DbSize)),
 			humanize.Bytes(uint64(status.Resp.DbSizeInUse)),
+			fmt.Sprint(100-(status.Resp.DbSizeInUse*100/status.Resp.DBSize)) + "%",
+			humanize.Bytes(uint64(status.Resp.DbSizeQuota)),
```

Comment:
removed this for the time being to see about clearing the checks but will want to add it in before merge pending the other PR that will be setting this value - will revisit.

Reply:
can you run the build and test locally first?